docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sphinx>=4.0
```

Comment:
Add them to dev section of poetry by below command. The requirements file is not necessary.

`poetry add --dev sphinx sphinx-argparse sphinx-rtd-theme sphinxemoji`